Assets/Thirdweb/Examples/Scripts/Prefabs/Prefab_ConnectWallet.cs

```diff
@@ -193,7 +194,7 @@ private void ValidateConnection(WalletProvider walletProvider)
                     _email = SupportedWalletsUI[WalletProvider.MagicLink].emailInput.text;
                 }
             }
-            else if (walletProvider == WalletProvider.LocalWallet)
+            else if (walletProvider is WalletProvider.LocalWallet or WalletProvider.LocalHdWallet)
```

Comment:
Prefer to be on the safe side here in case older unity versions don't support C# 9 syntax, if it doesn't break 2021 it should be ok